xmppserver/src/main/java/org/jivesoftware/openfire/spi/ConnectionListener.java

```diff
@@ -130,7 +130,7 @@ public ConnectionListener( ConnectionType type, String tcpPortPropertyName, int
         this.bindAddress = bindAddress;
         this.identityStoreConfiguration = identityStoreConfiguration;
         this.trustStoreConfiguration = trustStoreConfiguration;
-        this.compressionPolicyPropertyName = compressionPolicyPropertyName;
+        this.compressionPolicyPropertyName = compressionPolicyPropertyName; 
```

Comment:
gasp :)

Reply:
That's an unexpected change. IDE doing things?